service/src/test/resources/uat/output/TC4-9465701718_Guerra_full_20210119.xml

```diff
@@ -769,7 +780,7 @@ Quantity: 1750.000 mL
         </component>
             <component typeCode="COMP" contextConductionInd="true">
     <CompoundStatement classCode="CLUSTER" moodCode="EVN">
-        <id root="6423EA0F-1F1C-4255-AEBA-D142BE836D50"/>
+        <id root="1E5D6932-C778-4997-B624-8709D2FBB189"/>
```

Comment:
This empty CompoundStatement fails schema validation due to missing component/specimen

org.xml.sax.SAXParseException; lineNumber: 237; columnNumber: 25; cvc-complex-type.2.4.b: The content of element 'CompoundStatement' is not complete. One of '{"urn:hl7-org:v3":specimen, "urn:hl7-org:v3":component}' is expected.

See "skipped" UAT test for Guerra. Is it related to this story or something else?

Reply:
I think this one is covered in https://gpitbjss.atlassian.net/browse/NIAD-1470